wolfcrypt/test/test.c

```diff
@@ -14054,7 +14088,13 @@ static int rsa_test(void)
 
 #if (!defined(WOLFSSL_RSA_VERIFY_ONLY) || defined(WOLFSSL_PUBLIC_MP)) && \
                                  !defined(WC_NO_RSA_OAEP) && !defined(WC_NO_RNG)
+#if defined(WOLFSSL_RENESAS_CCRX) /* renesas CC-RX compiler deos not allow this macro*/
```

Comment:
This issue is the same one I encountered with Keil and the `DECLARE_VAR_INIT` macro. Consider reverting this and relying on the fixes in this PR. https://github.com/wolfSSL/wolfssl/pull/3670

Reply:
That's good news.